public/app/core/components/Select/FolderPicker.tsx

```diff
@@ -336,6 +341,7 @@ export function FolderPicker(props: Props) {
           onChange={onFolderChange}
           onCreateOption={createNewFolder}
           isClearable={isClearable}
+          width={42}
```

Comment:
We should not be setting the width explicitly because this breaks it in other places where this component is used, do we 100% want to change the width for alerting?

Reply:
true, let me check it 👀

Reply:
Removed ✅ @JoaoSilvaGrafana